SX1272/SX1272_LoRaRadio.cpp

```diff
@@ -1757,6 +1731,15 @@ void SX1272_LoRaRadio::handle_dio0_irq()
                         }
                     }
 
+                    // This block was moved from dio2_handler.
+                    // We can have a snapshot of RSSI here as at this point it
+                    // should be more smoothed out.
+                    _rf_settings.fsk_packet_handler.rssi_value = -(read_register(REG_RSSIVALUE) >> 1);
+                    _rf_settings.fsk_packet_handler.afc_value = (int32_t)(double)(((uint16_t)read_register(REG_AFCMSB) << 8) |
```

Comment:
Generally you should be trying to reduce or totally eliminate use of floating point, if you're aiming at low-end devices. At least use `float` rather than `double` given that a lot of chips we're on have single-precision hardware only.

Reply:
Yeah I will change everything with float precision. Good point.